truck_test_rapid_analyzer.py

```diff
@@ -0,0 +1,614 @@
+
+    space_columns(ws_data, 1)
+
+    if not single_workbook:
+        space_columns(ws_consts, 10, 1)
+        active_wb.save(output_path)
+
+
+
+if __name__ == "__main__":
+    print('Analyzer active')
+    
+    if FILE_INPUT_HITLIST:
+        hitlist_path = HITLIST_PATH
+        hitlist = []
+        with open(hitlist_path, 'r') as hitlist_file:
+            hitlist = list(hitlist_file)
+
+        prefixes_path = SINGLE_OUTPUT_SHEETNAMES_PATH
+        prefixes = []
+        with open(prefixes_path, 'r') as prefixes_file:
+            prefixes = list(prefixes_file)
+
+
+        if SINGLE_OUTPUT_FILE:
+            print('Exporting analysis to single file: '
+                  + SINGLE_OUTPUT_FILE_PATH)
+
+            global_workbook = WB()
+            ws_consts = global_workbook.create_sheet(title="Constants")
+            f_create_constants_table(ws_consts)
+            Consts().create_defined_names(global_workbook)
+            space_columns(ws_consts, 10, 1)
+
+
+            for target, prefix in zip(hitlist, prefixes):
+                execute_analysis(target[:-1],
+                                 output_path=target[:-1] + '___analyzed.xlsx',
+                                 single_file=True,
+                                 single_workbook=global_workbook,
+                                 sheetname_prefix=prefix,
+                                 condensed_version=CONDENSED_EXPORT_VERSION)
+
+                print(target[:-1] + " analyzed")
+
+
+            global_workbook.save(SINGLE_OUTPUT_FILE_PATH)
+            print('\n Analysis complete')
+        else:
+            for target in hitlist:
+                execute_analysis(target[:-1],
+                                 output_path=target[:-1] + '___analyzed.xlsx')
+
+                print(target[:-1] + " analyzed")
+
+            print('\n Analysis complete')
+    else:
+        if USE_HARD_CODED_PATH:
+            itarget = HARD_CODED_PATH
+            otarget = HARD_CODED_OUTPUT
+        else:
+            itarget = input("Enter the input target path")
+            otarget = input("Enter the output target path")
+
+        active_wb = WB()
+        ws_data = active_wb.create_sheet(title="Data")
+        ws_consts = active_wb.create_sheet(title="Constants")
+        ws_graphs = active_wb.create_sheet(title="Graphs")
+
+        f_create_constants_table(ws_consts)
+        create_headers(ws_data)
+
+        csvfile = open(itarget, newline='')
+        csvreader = csv.reader(csvfile, delimiter=',')
+
+        row_idx = 1
+        for row in list(csvreader):
+            if row_idx and row_idx != 1:
+                if simple_filter(row):  # Data Filtering
+                    process_sheet_row(row, row_idx, ws_data)
+                else:
+                    row_idx -= 1
+
+            row_idx += 1
+
+        create_graphs(ws_data, ws_graphs, row_idx)
+
+        space_columns(ws_data, 1)
+        space_columns(ws_consts, 10, 1)
+
+        active_wb.save(otarget)
```

Comment:
Might be nice to try and break the code up into smaller individual files and have them import each other. Jacob told me that you want the code in main to be as small as possible and everything else should be in its own little piece.

Over all though, your code was nice and very easy to review! Thanks for making it well written and easy to understand